packages/longhorn/longhorn.patch

```diff
@@ -513,248 +845,76 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/templates/crd
--    {{- else }}
--        image: "{{ .Values.image.longhorn.ui }}:{{ .Values.image.longhorn.uiTag }}"
--    {{- end }}
-+        image: {{ template "system_default_registry" . }}{{ .Values.image.longhorn.ui.repository }}:{{ .Values.image.longhorn.ui.tag }}
-         imagePullPolicy: IfNotPresent
-         securityContext:
-           runAsUser: 0
-diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/templates/postupgrade-job.yaml packages/longhorn/charts/templates/postupgrade-job.yaml
---- packages/longhorn/charts-original/templates/postupgrade-job.yaml
-+++ packages/longhorn/charts/templates/postupgrade-job.yaml
-@@ -17,11 +17,7 @@
-     spec:
-       containers:
-       - name: longhorn-post-upgrade
--    {{- if .Values.privateRegistry.registryUrl }}
--        image: "{{ .Values.privateRegistry.registryUrl }}/{{ .Values.image.longhorn.manager }}:{{ .Values.image.longhorn.managerTag }}"
--    {{- else }}
--        image: "{{ .Values.image.longhorn.manager }}:{{ .Values.image.longhorn.managerTag }}"
--    {{- end }}
-+        image: {{ template "system_default_registry" . }}{{ .Values.image.longhorn.manager.repository }}:{{ .Values.image.longhorn.manager.tag }}
-         imagePullPolicy: Always
-         command:
-         - longhorn-manager
-diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/templates/uninstall-job.yaml packages/longhorn/charts/templates/uninstall-job.yaml
---- packages/longhorn/charts-original/templates/uninstall-job.yaml
-+++ packages/longhorn/charts/templates/uninstall-job.yaml
-@@ -17,11 +17,7 @@
-     spec:
-       containers:
-       - name: longhorn-uninstall
--    {{- if .Values.privateRegistry.registryUrl }}
--        image: "{{ .Values.privateRegistry.registryUrl }}/{{ .Values.image.longhorn.manager }}:{{ .Values.image.longhorn.managerTag }}"
--    {{- else }}
--        image: "{{ .Values.image.longhorn.manager }}:{{ .Values.image.longhorn.managerTag }}"
--    {{- end }}
-+        image: {{ template "system_default_registry" . }}{{ .Values.image.longhorn.manager.repository }}:{{ .Values.image.longhorn.manager.tag }}
-         imagePullPolicy: Always
-         command:
-         - longhorn-manager
-diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/values.yaml packages/longhorn/charts/values.yaml
---- packages/longhorn/charts-original/values.yaml
-+++ packages/longhorn/charts/values.yaml
-@@ -1,16 +1,37 @@
- # Default values for longhorn.
- # This is a YAML-formatted file.
- # Declare variables to be passed into your templates.
-+global:
-+  cattle:
-+    systemDefaultRegistry: ""
-+
- image:
-   longhorn:
--    engine: longhornio/longhorn-engine
--    engineTag: v1.0.2
--    manager: longhornio/longhorn-manager
--    managerTag: v1.0.2
--    ui: longhornio/longhorn-ui
--    uiTag: v1.0.2
--    instanceManager: longhornio/longhorn-instance-manager
--    instanceManagerTag: v1_20200514
-+    engine:
-+      repository: rancher/longhornio-longhorn-engine
-+      tag: v1.0.2
```

Comment:
None of these images have changed, is that not required for the release ?

Reply:
This means no extra modification in `values.yaml` is needed for the new release.